tests/devapps/MSI/HelperService/MSIHelperService/AzureFunction/GetEnvironmentVariables.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+//#r "Newtonsoft.Json" //uncomment this on the function app 
```

Comment:
Isn't this for the function app?

Reply:
Yes, this code is deployed on the function app. commented that function app line so the project can build